NAMESPACE

```diff
@@ -121,22 +117,33 @@ import(circlize)
 import(config)
 import(cowplot)
 import(data.table)
-import(dbplyr)
 import(digest)
 import(dplyr)
+import(forcats)
 import(g3viz)
 import(ggbeeswarm)
 import(ggplot2)
 import(ggpubr)
 import(ggrepel)
 import(ggsci)
 import(ggthemes)
+import(glue)
 import(grid)
+import(gridExtra)
 import(maftools)
 import(metaviz)
 import(parallel)
 import(plotly)
+import(purrr)
+import(readr)
+import(reshape2)
```

Comment:
Which function imports reshape2? I just dropped it in the latest mster update but seems like it is used somewhere else, too?

Reply:
Yes, it's imported by `prettyForestPlot`.

Reply:
I think your branch is not up to date with master. This was re-worked to remove the reshape2 dependency. The same also applies to stats package - in the recent master it is no longer imported

Reply:
The `stats` package is a default R package and is available when R starts. Therefore, to use `p.adjust`, we do not need to refer to the package specifically. Including the `stats`  in NAMESPACE explicitly only created name conflicts and many warnings generated once GAMBLR is loaded.

Reply:
You could always update the code to say `stats::p.adjust()`	when it's called or add stats to the @imports line, right?

Reply:
Ok, thanks for clarifying! Makes sense.

Reply:
FYI, after educating myself on what packages are default R packages (i.e loaded every time you start R), I found two additional packages that can be removed for the same reasons as `utils` and `stats`. These packages are `parallel` and `grid`.  I have removed them from being imported by any functions (i/e preventing them to populate NAMESPACE).

Reply:
Thanks for checking!